plans/step9-plan.md

```
@@ -0,0 +1,283 @@
+   - 기대 결과:
+     - `/admin/login`에서 1차/2차 인증 성공 후 `/admin/write` 접근 가능
+     - 글 게시 후 `/posts/{slug}`로 이동하고 본문/태그가 노출됨
+
+12. **`/write` 하위 호환 리다이렉트 검증 (`?id` 보존 포함)**
+   ```bash
+   npm run test:ui -- tests/ui/admin-write-redirect.spec.ts
+   ```
+   - 기대 결과:
+     - `/write` 접근 시 `/admin/write`로 리다이렉트
+     - `/write?id=123` 접근 시 `/admin/write?id=123`로 리다이렉트
+
```

markdownlint-cli2 (0.20.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)